docs/ui/workspaces.md

```diff
@@ -0,0 +1,104 @@
+---
+description: Workspaces are isolated environments for flows and deployments within Prefect Cloud.
+tags:
+    - UI
+    - Prefect Cloud
+    - workspaces
+    - deployments
+---
+
+# Workspaces <span class="badge cloud"></span>
+
+A workspace is an isolated environment within Prefect Cloud for your flows and deployments. Workspaces are only available to Prefect Cloud accounts.
+
+Workspaces could be used in any way you like to organize or compartmentalize your workflows. For example, you could use separate workspaces to isolate dev, staging, and prod environments, or to provide separation between different teams.
+
+The number of available workspaces varies by [Prefect Cloud plan](https://www.prefect.io/pricing/). See [Pricing](https://www.prefect.io/pricing/) if you need additional workspaces or users.  
+
+### Workspaces overview
+
+When you first log into Prefect Cloud, you will be prompted to create your own initial workspace. After creating your workspace, you'll be able to view flow runs, flows, deployments, and other workspace-specific features in the Prefect Cloud UI.
+
+![Viewing a workspace dashboard in the Prefect Cloud UI.](/img/ui/cloud-new-workspace.png)
+
+Select the **Prefect Logo** in the left navigation area to see all of the workspaces you can access. 
```

Comment:
We will likely change the landing page for the Prefect Logo at some point to a more intentionally-built homepage. So when we have the opportunity like this, I think we should direct users to the "4-squares" item in the left-nav area, which is dedicated to the workspaces page (and will reliably do so in the future).